### PR TITLE
Document automatic signing of UWP apps

### LINKED ISSUE #3399: Adjust UWP export documentation to include automatic code signing setup.
**Your Godot version:**
Godot 3.2.1.stable Win64 non-mono

**Issue description:**
The documentation was great and I was able to easily get everything working as described. During this exercise, I felt that Godot was asking my to do more than I should have to per export. After filing the following proposal, it was suggested that I use the Editor Settings. This was not mentioned in the documentation and thought it would make new comer's lives easier to know they don't need to manually run or script a signtool.exe invocation outside of Godot for each UWP export.

Here is the above mentioned proposal ticket: 
[https://github.com/godotengine/godot-proposals/issues/721](https://github.com/godotengine/godot-proposals/issues/721)

I recommend that:
* The documentation should provide references to the Editor Settings / Export / UWP for setting up the signtool path and other relevant settings.
* Replace "Currently, Godot *does not support signing* of packages, so you need to use external tools to do so." with something like "Currently, Godot depends on external tools setup in editor settings to perform signing of packages." My reasoning is that Godot *does support signing*, its just not directly linked into or inline-ed into the engine itself.

**URL to the documentation page:**
[https://docs.godotengine.org/en/3.2/getting_started/workflow/export/exporting_for_uwp.html](https://docs.godotengine.org/en/3.2/getting_started/workflow/export/exporting_for_uwp.html)

### PR DESCRIPTION
**Pull request by skyace65**
Documents how to have UWP apps automatically signed on export. All images have gone through png compression. Closes #3399

### CODE REVIEW

File: getting_started/workflow/export/exporting_for_uwp.rst [hunk 1]
```diff
@@ -8,8 +8,8 @@ installed as a Windows App or submitted to the Windows Store. Exporting UWP
 packages also works from any platform, not only from Windows.
 
 However, if you want to install and run the app, you need to sign it with a
-trusted signature. Currently, Godot does not support signing of packages, so you
-need to use external tools to do so.
+trusted signature. Currently, Godot supports automatic signing of packages with
```
Comment (mhilbrunner) [inline getting_started/workflow/export/exporting_for_uwp.rst:11]:
```suggestion
trusted signature. Godot supports automatic signing of packages with
```

File: getting_started/workflow/export/exporting_for_uwp.rst [hunk 2]
```diff
@@ -60,10 +60,26 @@ app. Open the Command Prompt as Administrator and run the following command::
 
     Certutil -addStore TrustedPeople MyKey.cer
 
-Signing the package
--------------------
+Setting up automatic signing
+-----------------------------
```
Comment (mhilbrunner) [inline getting_started/workflow/export/exporting_for_uwp.rst:64]:
```suggestion
Setting up automatic signing
----------------------------
```

File: getting_started/workflow/export/exporting_for_uwp.rst [hunk 3]
```diff
@@ -60,10 +60,26 @@ app. Open the Command Prompt as Administrator and run the following command::
...
 
-Finally, use ``SignTool.exe`` from the Windows SDK or Visual Studio::
+To setup automatic signing on export you need to go to Editor Settings > Export > Uwp.
+From there you need to click on the folder for ``Signtool``, and navigate to
+the SignTool.exe file on your computer.
```
Comment (mhilbrunner) [inline getting_started/workflow/export/exporting_for_uwp.rst:68]:
```suggestion
To setup automatic signing on export you need to go to Editor Settings > Export > UWP.
From there you need to click on the folder for ``Signtool``, and navigate to
the ``SignTool.exe`` file on your computer.
```

File: getting_started/workflow/export/exporting_for_uwp.rst [hunk 4]
```diff
@@ -60,10 +60,26 @@ app. Open the Command Prompt as Administrator and run the following command::
...
+After setting that up close the editor settings, go to Project > Export,
+and select the UWP preset. Under the ``Signing`` options click on the folder
+next to ``Certificate`` and go to the certificate file. Then enter the
+pfxPassword in the password field.
+
+.. image:: img/UWP_export_signing.png
+
+Your project will now be automatically signed on export.
+
+If you want to sign an exported app manually run SignTool.exe and use the
```
Comment (mhilbrunner) [inline getting_started/workflow/export/exporting_for_uwp.rst:81]:
```suggestion
If you want to sign an exported app manually run ``SignTool.exe`` and use the
```

**Comment by mhilbrunner** [comment]
Left a few notes, otherwise looks good :)

**Comment by skyace65** [comment]
I've made those fixes but `Uwp` isn't a mistake, that's exactly how it appears in the editor settings.

**Comment by mhilbrunner** [comment]
That's fine then :) Thank you!

**Pull request merged**